Dockerfile

```diff
@@ -1,30 +1,17 @@
-FROM fedora:32
+FROM node:16-alpine
+
+RUN apk --no-cache --update add graphicsmagick python3 bash \
+    git \
+    build-base \
+    g++ \
+    cairo-dev \
+    jpeg-dev \
+    pango-dev \
+    giflib-dev
 
 RUN mkdir -p /opt/node-markup
 WORKDIR /opt/node-markup
 
-RUN curl -sL -o node.rpm https://rpm.nodesource.com/pub_6.x/fc/26/x86_64/nodesource-release-fc26-1.noarch.rpm \
- && rpm -ivh node.rpm \
- && dnf remove -y nodejs npm || true \
- && dnf install -y nodejs
-
-RUN npm config set umask 002 \
- && npm config set unsafe-perm true \
- && npm install -g npm@6.13.7 strip-ansi@3.0.1 && npm version
-
-RUN npm build /usr/lib/node_modules/*
-
-RUN dnf -y install \
- python \
- GraphicsMagick \
- gcc-c++ \
- perl-Digest-SHA \
```

Comment:
I'm assuming things listed here that aren't listed above (like perl-Digest-SHA) just aren't needed anymore?

Reply:
Yep! Everything seems to work decently well on `node:16-alpine`. Though these `dnf install`s have been swapped out with analogous `apk add`s up the Dockerfile a bit.